tests/utils/helpers.rs

```diff
@@ -750,12 +775,24 @@ impl TestWallet {
         let params = TransferParams::with(fee, sats);
         let (mut psbt, _psbt_meta, consignment) = self.wallet.pay(&invoice, params).unwrap();
 
+        let mut cs_path = self.wallet_dir.join("consignments");
+        std::fs::create_dir_all(&cs_path).unwrap();
+        cs_path.push(consignment.consignment_id().to_string());
+        cs_path.set_extension("yaml");
+        let mut file = std::fs::File::options()
+            .read(true)
+            .write(true)
+            .create_new(true)
+            .open(cs_path)
+            .unwrap();
```

Comment:
BTW we can use much shorter form for this like `fs::File::create_new`.

Reply:
by using that I receive the following warning:
```
warning: current MSRV (Minimum Supported Rust Version) is `1.75.0` but this item is stable since `1.77.0`
```
therefore thanks for the suggestion but I will keep it as it is